openshift-knative-operator/pkg/serving/istio.go

```diff
@@ -0,0 +1,16 @@
+package serving
+
+import (
+	mf "github.com/manifestival/manifestival"
+	"github.com/openshift-knative/serverless-operator/openshift-knative-operator/pkg/common"
+	corev1 "k8s.io/api/core/v1"
+	operatorv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
+)
+
+func enableSecretInformerFiltering(ks operatorv1alpha1.KComponent) mf.Transformer {
+	if v, ok := ks.GetAnnotations()["knative.dev/enable-secret-informer-filtering-by-cert-uid"]; ok {
```

Comment:
```suggestion
	if v, ok := ks.GetAnnotations()["serverless.openshift.io/enable-secret-informer-filtering"]; ok {
```

The `knative.dev` is upstream's domain so we should use ``serverless.openshift.io`.
And `enable-secret-informer-filtering-by-cert-uid` is too long, isn't it :sweat_smile:

Reply:
I dont see a big difference but wfm.

Reply:
@nak3 new name is: `serverless.openshift.io/enable-secret-informer-filtering`, looks good?